scipy/interpolate/tests/test_interpolate.py

```diff
@@ -521,6 +521,22 @@ def test_complex_coef(self):
                 assert_allclose(p(xp, nu).real, p_re(xp, nu))
                 assert_allclose(p(xp, nu).imag, p_im(xp, nu))
 
+    def test_axis(self):
+        np.random.seed(12345)
+        c = np.random.rand(3, 4, 5, 6, 7, 8)
+        c_s = c.shape
+        xp = np.random.random((1, 2))
+        for axis in (0, 1, 2, 3):
```

Comment:
``axis=4,5`` also need to be tested. Doesn't work just inserting 4,5 here, so another test with ``axis=-1`` is also fine.